dxchange/exchange.py

```diff
@@ -157,7 +158,8 @@ def read_als_832(fname, ind_tomo=None, normalized=False):
         ind_dark = list(range(0, ndark))
 
     # Read image data from tiff stack.
-    tomo = dxreader.read_tiff_stack(tomo_name, ind=ind_tomo, digit=4)
+    tomo = dxreader.read_tiff_stack(tomo_name, ind=ind_tomo, digit=4,
+                                    slc=(sino, None))
```

Comment:
@tacaswell  it does smell bad ... thank you for point it out. For some reason we are flipping the order between datasets consisting of multiple files  (tiff, xrm)  vs  single files (hdf, netcdf etc.). We should make it consistent.